.github/workflows/ci-publish-docs-branch.yml

```diff
@@ -20,15 +37,52 @@ jobs:
 
       - name: Create and checkout new docs branch
         run: |
-          git config --global user.name "github-actions[bot]"
-          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions"
+          git config --global user.email "github-actions@github.com"
           git checkout -b docs
 
       - name: Generate docs
         shell: pwsh
         run: |
           Import-Module ./BenchPress/Helpers/BenchPress.Azure/BenchPress.Azure.psd1
-          New-MarkdownHelp -Module BenchPress.Azure -OutputFolder ./docs
+
+          New-Item -ItemType Directory -Path "./docs/helpers" -Force
+
+          New-MarkdownHelp -Module BenchPress.Azure -OutputFolder ./docs/helpers -Force -AlphabeticParamsOrder -UseFullTypeName -NoMetadata
+
+          foreach ($File in Get-ChildItem ./docs/helpers/*.md) {
+            Write-Host "Updating $File"
+            # Add front matter
+            $Frontmatter = ""
+            $FrontMatter += "---`r`n"
+            $FrontMatter += "WARNING: `"PLEASE DO NOT MODIFY THIS FILE, IT'LL BE OVERWRITTEN`"`r`n"
+            $FrontMatter += "layout: `"../../layouts/main.astro`"`r`n"
+            $FrontMatter += "title: `"$($File.BaseName)`"`r`n"
+            $FrontMatter += "module: `"BenchPress.Azure`"`r`n"
+            $FrontMatter += "version: `"${{ github.sha }}`"`r`n"
+            $FrontMatter += "updatedOn: `"$(Get-Date -Format "yyyy-MM-dd HH:mm K")`"`r`n"
+            $FrontMatter += "---`r`n"
+            # Fix escaping issues
+            $Content = Get-Content $File -Raw
+            $Content = $Content -Replace '\\`', '`'
+            $Content = $Content -Replace "\\\[", "["
+            $Content = $Content -Replace "\\\]", "]"
+            # Fix syntax highlighting
+            $Lines = $Content -Split "`r`n"
```

Comment:
Can we break this up into multiple functions for readability and testability? The current find replace approach for fixing markdown errors ties us heavily to the conventions used in the comments (eg the syntax keyword.)